DigitalLearningSolutions.Web/ViewModels/SuperAdmin/Faqs/FaqsPageViewModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.SuperAdmin.Faqs
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Web.Models.Enums;
+    using DigitalLearningSolutions.Web.ViewModels.Shared.Faqs;
+
+    public class FaqsPageViewModel
+    {
+        public FaqsPageViewModel(IEnumerable<SearchableFaq> faqs)
+        {
+            Faqs = faqs.Select(f => new SearchableFaqViewModel(f));
+            CurrentPage = SuperAdminSystemPage.FAQs;
+        }
+
+        public IEnumerable<SearchableFaqViewModel> Faqs { get; set; }
+        public SuperAdminSystemPage CurrentPage { get; set; }
```

Comment:
This can be simplified further with
`        public SuperAdminSystemPage CurrentPage => SuperAdminSystemPage.FAQs;`
so we don't even need to set it in the constructor.